src/components/common.js

```diff
@@ -25,6 +25,26 @@ const styles = {
     display: 'inline-flex', justifyContent: 'space-around', alignItems: 'center', height: '2.25rem',
     fontWeight: 500, fontSize: 14, textTransform: 'uppercase', whiteSpace: 'nowrap',
     userSelect: 'none'
+  },
+  tabBar: {
+    container: {
+      display: 'flex', alignItems: 'center',
+      fontWeight: 400, textTransform: 'uppercase',
+      height: '3.75rem',
+      borderBottom: `1px solid ${terraSpecial()}`, flex: ''
+    },
+    tab: {
+      flex: 'none', outlineOffset: -4, marginRight: '1rem',
+      alignSelf: 'stretch', display: 'flex', justifyContent: 'center', alignItems: 'center',
+      borderBottomWidth: 8, borderBottomStyle: 'solid', borderBottomColor: 'transparent'
+    },
+    active: {
+      borderBottomColor: terraSpecial(),
+      fontWeight: 600
+    },
+    hover: {
+      fontWeight: 600
```

Comment:
TOL: Changing between normal and bold text changes the width very slightly, which can cause the line to shift around on hover. We have a way of dealing with this, but it's slightly tricky and doesn't need to be included in this PR since this isn't actually being used yet.